packages/data-stores/src/reader/queries/use-site-subscriptions-query.ts

```diff
@@ -91,11 +91,13 @@ const useSiteSubscriptionsQuery = ( {
 			}
 		);
 
+	const nextPage = hasNextPage && ! isFetching && data ? data.pages.length + 1 : null;
```

Comment:
Neat optimization 👍 

But are we really using the calculated `nextPage` value for anything here? It seems like it could have been a simpler value (`true`) since we only use it as a boolean anyway.

Reply:
It's necessary to maintain the functionality to refetch pages, it needs to change so that the `useEffect` which its used by runs again.